docs/common_tasks.rst

```diff
@@ -51,6 +51,42 @@ Running the tests
 
      (venv)vagrant@precise32:~/treeherder$ flake8
 
+
+Look up credentials for a project
+---------------------------------
+
+To submit data to an existing project, you need corresponding OAuth credentials, even if you're submitting to your local server.
+
+Prerequisite: you must have previously exported credentials for all data source projects. This is part of the usual Treeherder setup process.
+
+  .. code-block:: bash
+
+      > ./manage.py export_project_credentials
+
+The above command writes all project credentials to ``treeherder/etl/data/
+credentials.json`` for use by the Treeherder service.
```

Comment:
Could you move this up to the previous line? I think this is an unintentional line break.

And yes, this is great. I didn't know how to get the oauth credentials directly before and now I do. :)